features/admin.users.v1/components/wizard/steps/add-user-basic.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.truncate {
```

Comment:
WDYT about wrapping these classes with a parent class (ex: `.add-user-basic`) to scope these to the AddUserBasic component?

Reply:
Fixed with 5cbff8089174971c91a1c1aabd4c1a11270bdb83